packages/core-utils/src/route.ts

```diff
@@ -184,6 +184,7 @@ function getSortValues(
 const modeComparatorValue = {
   SUBWAY: 1,
   TRAM: 2,
+  TROLLEYBUS: 9,
```

Comment:
Wait shouldn't this be 11? In the GTFS spec trolleybus is 11, doesn't OTP follow the spec

Reply:
I don't think this has anything to do with OTP mode numbers. This has to do with determining route sort order